Created a solution pack -  SOC Overview)

cs-yash-patodkar opened the pull request:
Problem Statement:
Currently, we need to manually install and configure SOC management widget.Expected: We can create new Solution Pack with below requirements
1. Install SOC Management widget
2. Configure SOC visualisation Dashboard with default values :
Where - 'Created On' Last 7 Days

Description :
The Solution contains a Dashboard which has widget(Soc Management)
The Solution pack will install the soc management Widget if not already installed and create Dashboard With the Widget

Test Case:
1. Installed the SP
2. The Widget gets installed 
3. A dashboard is created with the widget


PMDB : https://pmdb.fortinet.com/ProjectManagement/viewDocument.php?id=21297

cs-yash-patodkar committed:
Created a solution pack(Solution Pack SOC Overview) with a dashboard that includes a widget(Soc Management)

cs-sujeet-ahirrao commented on dashboards/SOC Management Dashboard.json:
```diff
@@ -0,0 +1,40 @@
+    "displayName": "SOC Management Dashboard",
+    "type": "rows",
+    "templateType": "dashboard",
+    "roles": [],
```
Add Roles to the Dashboard

cs-yash-patodkar replied:
Added roles, Full app permission, SOC analyst, SOC manager !

cs-yash-patodkar committed:
Added Roles To the dashboard

Roles added :
SOC Analyst
SOC Manager
Full App Permission

cs-sujeet-ahirrao commented on README.md:
```diff
@@ -0,0 +1,19 @@
+
+Use absolute links
+
+| [Installation](https://github.com/fortinet-fortisoar/solution-pack-soc-overview/blob/release/1.0.0/docs/setup.md#installation) | [Configuration](https://github.com/fortinet-fortisoar/solution-pack-soc-overview/blob/release/1.0.0/docs/setup.md#configuration) | [Usage](https://github.com/fortinet-fortisoar/solution-pack-soc-overview/blob/release/1.0.0/docs/usage.md) | [Contents](https://github.co ...
```
These links are not working

cs-yash-patodkar replied:
Resolved the error, updated the links

cs-yash-patodkar committed:
Updated the install, configure, usage links

The install, configure, usage links were not working, gave a relative path

cs-yash-patodkar committed:
Updated the Home Link

cs-yash-patodkar committed:
Renamed us to contents.md

cs-yash-patodkar committed:
Changed Home Link

cs-yash-patodkar committed:
Changed Home Link

cs-suyog-jinturkar commented on README.md:
```diff
@@ -0,0 +1,19 @@
+* **Version**: 1.0.0
+* **Certified**: No
+* **Publisher**: Fortinet
+* **Compatible Version**: FortiSOAR v7.2.2
```
Why 7.2.2 only?

cs-suyog-jinturkar replied:
info.json has value --> "fsrMinCompatibility": "7.2.0",

cs-yash-patodkar replied:
The SOC management widget has a minimum compatibility of 7.2.2

cs-suyog-jinturkar reviewed (commented):
Added one small comment

cs-yash-patodkar committed:
Changed Minimum compatibility

Minimum compatibility changed from 7.2.0 to 7.2.2 since the widget has a minimum compatibility of 7.2.2
